streams/src/main/java/org/apache/kafka/streams/state/internals/SessionKeySchema.java

```diff
@@ -97,4 +97,19 @@ public boolean hasNext(final KeyValueIterator<Bytes, ?> iterator) {
         return segments.segments(from, Long.MAX_VALUE);
     }
 
+    @Override
```

Comment:
I don't see why we need to add the next three methods to the `KeySchema` interface? They are the same in each case.